docs/production-deployment/cloud/get-started/billing-and-cost.mdx

```diff
@@ -51,7 +51,7 @@ This balance adjusts with use and appears on the first line of your Invoices tab
 :::note Billing Cycles
 
 Billing cycles normally begin on the first of the month (UTC).
-The minimum support fee for your first month is prorated based on your sign-up date.
+The minimum plan fee for your first month is prorated based on your sign-up date.
```

Comment:
"plan fee" is a more correct term because it not only enables a certain support tier, but also it unlocks higher included actions/storage + advanced features (e.g. SCIM)